pkg/release/release.go

```diff
@@ -36,3 +36,21 @@ func GetComponentVersionsFromRelease(ctx context.Context, ctrlReader client.Read
 
 	return ret, nil
 }
+
+// ContainsAzureOperator checks if the specified release contains azure-operator.
+//
+// In order to perform the check, this function is calling GetComponentVersionsFromRelease function,
+// which is caching obtained components in memory. See GetComponentVersionsFromRelease docs for
+// more info about the caching.
```

Comment:
Is this comment valid when the caching actually happens in the passed client which can be either caching one or the "normal" one (same interface)?

Reply:
True, this is not needed anymore.